scripts/linters/js_ts_linter.py

```diff
@@ -46,6 +47,12 @@
 # pylint: enable=wrong-import-order
 # pylint: enable=wrong-import-position
 
+FILES_EXCLUDED_FROM_ANY_TYPE_CHECK_PATH = os.path.join(
+    CURR_DIR, 'scripts', 'linters', 'excluded_any_type_files.json')
+
+FILES_EXCLUDED_FROM_ANY_TYPE_CHECK = json.load(open(
+    FILES_EXCLUDED_FROM_ANY_TYPE_CHECK_PATH, 'r'))
```

Comment:
Does the file get closed correctly using this? Shouldn't we still use `with python_utils.open_file(EXCLUDE_ANY_TYPE_FILES_PATH, 'r') as f` and then pass `f` into `json.load`?

Reply:
Done